hermes/src/price_feeds.rs

```diff
@@ -0,0 +1,92 @@
+//! Price Feeds
```

Comment:
I think it is more suitable to move this inside the `networks/pythnet.rs` as it is kind of the adapter to the Pythnet network.